packages/fuels-contract/Cargo.toml

```diff
@@ -27,3 +27,7 @@ strum_macros = "0.21"
 thiserror = { version = "1.0.26", default-features = false }
 tokio = "1.12"
 tracing = "0.1"
+
+[dev-dependencies]
+fuels = { path = "../fuels" }
```

Comment:
try to not import `fuels` but rather just the `fuels-test-helper`, because I think it's creating circular deps. Even if it's not, It's better to use the smallest import set anyway.